src/utils/layout.sh

```diff
@@ -10,9 +10,7 @@ rotate() {
   have=${have:1:${#have}-2}
   angle=$3
 
-  if [[ "$have" != "$want" ]]; then
-    bspc node "$node" -R "$angle"
-  fi
+  [ "$have" = "$want" ] || bspc node "$node" -R "$angle"
```

Comment:
i find `[ "$have" != "$want" ] &&` clearer than `[ "$have" = "$want" ] ||`, but that's quite a personal feeling :yum:

Reply:
Yes me too. I just change cause is feel wrong to me but ill change it back 👍🏻

Reply:
:+1: